### ISSUE TITLE
What is annotation files in test-folder??

### ISSUE THREAD

**Issue by edwardcho** [hendrycks/anomaly-seg#8]
Hello Sir,

I saw annotation files in test-folder.
What is the purpose of this file??
(When training, Is this file used??)

Thanks.

Edward Cho.

**Comment by xksteven** [comment]
I'll update the README file to make this distinction more clear. 

Within CARLA their are several towns that have been created so we use towns 1-3 for the purposes of training, town 4 for validation and finally for our "test" we use towns 5 and 6.  We have inserted the anomalies and only use this at test time to test how well the models can detect OOD objects in segmentation models.

**Comment by edwardcho** [comment]
Hello Sir,

I saw some Figure on your paper. (paper title : A Benchmark for Anomaly Segmentation).

Figure 1 on this paper.
![image](https://user-images.githubusercontent.com/29569662/85347650-24e2ed80-b534-11ea-935c-4eb9c1866eae.png)
I did want to know "Ground Truth" image when test.
As you means, this file is only for testing (back-data file for checking of performance OOD), this file did not use on training. Right?

Thanks,

Edward Cho.
